Scripts/developer_scripts/run_testsuite_with_cmake

```diff
@@ -319,9 +325,14 @@ run_test_on_platform()
   TEST_REPORT="TestReport_${CGAL_TESTER}_${PLATFORM}"
   mkdir -p Installation
   chmod 777 Installation
-  cp "${CGAL_BINARY_DIR}/package_installation.log" "Installation/${TEST_REPORT}"
+  cat "${CGAL_BINARY_DIR}/package_installation.log" >> "Installation/${TEST_REPORT}"
   
+  #call the python script to complete the results report.
+  python3 ${CGAL_DIR}/${TESTSUITE_DIR}test/get_deactivated_packages_info.py Installation/${TEST_REPORT} ${TEST_REPORT} results_${CGAL_TESTER}_${PLATFORM}.txt
```

Comment:
I do not understand the name of the script: `get_deactivated_packages_info.py`. Now that the script is finished, you can probably find a better name for it.